DESCRIPTION

```diff
@@ -1,10 +1,10 @@
 Package: bpmnVisualization
 Type: Package
-Title: BPMN Visualization - R Package
+Title: BPMN Visualization for R
```

Comment:
Possible titles
- Display BPMN Diagrams and execution data

Reply:
```suggestion
Title: Display a process on BPMN diagram with execution data
```

Reply:
Final proposal agreed with @csouchet

```suggestion
Title: Visualize process execution data on BPMN diagrams
```